app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -29,28 +29,80 @@ export default function NewNoteForm({ onClose, refetch }) {
         description: yup.string().required("Description is required")
       })}
     >
-      {({ isSubmitting }) => (
+      {() => (
         <Form>
-          <Input label="Title" name="title" className="mb-6" />
-          <Textarea label="Description" name="description" rows={8} />
-          <div className="nui-pane__footer nui-pane__footer--absolute">
-            <Button
-              onClick={onClose}
-              label="Cancel"
-              size="large"
-              style="secondary"
+          <Input
+            label="Title"
+            name="title"
+            className="mb-6"
+            required={true}
+            placeholder="Enter Note Title"
+          />
+          <Textarea
+            label="Description"
+            name="description"
+            rows={1}
+            required={true}
+            placeholder="Enter Note Description"
+          />
+          <div className="my-6">
+            <Select
+              isClearable
+              isSearchable
+              required={true}
+              label="Assigned Contact"
+              name="role"
+              options={[
+                {
+                  label: "Value One",
+                  value: "value1"
+                },
+                {
+                  label: "Value Two",
+                  value: "value2"
+                },
+                {
+                  label: "Value Three",
+                  value: "value3"
+                },
+                {
+                  label: "Value Four",
+                  value: "value4"
+                },
+                {
+                  label: "Value Five",
+                  value: "value5"
+                }
+              ]}
+              placeholder="Select a Role"
             />
-
-            <Button
-              type="submit"
-              label="Submit"
-              size="large"
-              style="primary"
-              className="ml-2"
-              disabled={isSubmitting}
-              loading={isSubmitting}
+          </div>
+          <div className="my-6">
+            <Select
+              isClearable
+              isSearchable
+              required={true}
+              isMulti
+              label="Tags"
+              name="tags"
+              options={[
+                {
+                  label: <Tag color="green" label="Tag One" size="small" />,
+                  value: "Tag One"
+                },
+                {
+                  label: <Tag color="blue" label="Tag Two" size="small" />,
+                  value: "Tag Two"
+                },
+                {
+                  label: <Tag color="red" label="Tag Three" size="small" />,
+                  value: "Tag Three"
+                }
+              ]}
+              placeholder="Select Role"
             />
           </div>
```

Comment:
Please use `space-y-*` utils from tailwindCSS rather than adding `mb-6` to each of the blocks individually.